src/build/sorter.ts

```diff
@@ -0,0 +1,67 @@
+import { readFileSync } from "fs";
+import { parse, ParseError, visit } from "jsonc-parser";
+import { URL } from "url";
+
+// Recursively sort object keys
+function sortObjectKeys<T>(obj: T): T {
+  if (obj && typeof obj === "object" && !Array.isArray(obj)) {
+    const sorted: Record<string, unknown> = {};
+    for (const key of Object.keys(obj).sort()) {
+      sorted[key] = sortObjectKeys((obj as any)[key]);
+    }
+    return sorted as T;
+  }
+  return obj;
+}
+
+// Check if the content has trailing commas
+function hasTrailingCommas(content: string): boolean {
+  let foundTrailingComma = false;
+
+  visit(content, {
+    onObjectEnd: (offset) => {
+      const lastChar = content[offset - 1];
+      if (lastChar === ",") {
+        foundTrailingComma = true;
+      }
+    },
+    onArrayEnd: (offset) => {
+      const lastChar = content[offset - 1];
+      if (lastChar === ",") {
+        foundTrailingComma = true;
+      }
+    },
+  });
+
+  return foundTrailingComma;
+}
+
+export function sortFiles(): void {
```

Comment:
It's not about sorting anymore, it's about verifying it's sorted. And it's not sorting "files", it's sorting fields.